### DISCUSSION TITLE
Flat ride previews are cutoff in the ride management window

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#26713] [Feature requests]
### Operating System

Windows 11 Pro

### OpenRCT2 build

OpenRCT2, v0.4.8-73-gefb6d48 (efb6d48 on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

The bug also exists in the original game (RCT2/RCTC)

### Describe the issue

The ride preview in the ride management window should be made taller. Suggested by @Gymnasiast

### Steps to reproduce

Place a flat ride and then proceed to check the ride vehicle window.

### Attachments

![asddasadsdasdsa](https://github.com/OpenRCT2/OpenRCT2/assets/94884086/e20883e2-670a-4dca-9857-25b4b18ffd41)

**Comment by MASHzj** [comment]
Related to this, I would also like it if that image was scrollable when there are too many vehicles to fit in. That would make it easier to have an overview when I use different colours per vehicle.

**Comment by EricWittrock** [comment]
Can I work on this issue?

**Comment by 733737** [comment]
sure! Nobody else has picked it up yet.

**Comment by Gymnasiast** [comment]
This is working as intended and not a bug. Moving this to Feature Requests.
